python/cuml/feature_extraction/_vectorizers.py

```diff
@@ -602,15 +639,245 @@ def get_feature_names(self):
+        Typically the delimiting character between words is a good choice.
+
+    Examples
+    --------
+    >>> from cuml.feature_extraction.text import HashingVectorizer
+    >>> corpus = [
+    ...     'This is the first document.',
+    ...     'This document is the second document.',
+    ...     'And this is the third one.',
+    ...     'Is this the first document?',
+    ... ]
+    >>> vectorizer = HashingVectorizer(n_features=2**4)
+    >>> X = vectorizer.fit_transform(corpus)
+    >>> print(X.shape)
+    (4, 16)
+    See Also
+    --------
+    CountVectorizer, TfidfVectorizer
+    """
+
+    def __init__(
+        self,
+        input=None,
+        encoding=None,
+        decode_error=None,
+        strip_accents=None,
+        lowercase=True,
+        preprocessor=None,
+        tokenizer=None,
+        stop_words=None,
+        token_pattern=None,
+        ngram_range=(1, 1),
+        analyzer="word",
+        n_features=(2 ** 20),
+        binary=False,
+        norm="l2",
+        alternate_sign=True,
+        dtype=cp.float32,
+        delimiter=" ",
+    ):
+        self.preprocessor = preprocessor
+        self.analyzer = analyzer
+        self.lowercase = lowercase
+        self.stop_words = stop_words
+        self.n_features = n_features
+        self.ngram_range = ngram_range
+        self.binary = binary
+        self.norm = norm
+        self.alternate_sign = alternate_sign
+        self.dtype = dtype
+        self.delimiter = delimiter
+
+        if dtype not in CUPY_SPARSE_DTYPES:
+            msg = f"Expected dtype in {CUPY_SPARSE_DTYPES}, got {dtype}"
+            raise ValueError(msg)
+
+        if self.norm not in ("l1", "l2", None):
+            raise ValueError(f"{self.norm} is not a supported norm")
+
+        sklearn_params = {
+            "input": input,
+            "encoding": encoding,
+            "decode_error": decode_error,
+            "strip_accents": strip_accents,
+            "tokenizer": tokenizer,
+            "token_pattern": token_pattern,
+        }
+        self._check_sklearn_params(analyzer, sklearn_params)
+
+    def partial_fit(self, X, y=None):
+        """
+        Does nothing: this transformer is stateless.
+        This method is just there to mark the fact that this transformer
+        can work in a streaming setup.
+        Parameters
+        ----------
+        X : cudf.Series(A Series of string documents).
+        """
+        return self
+
+    def fit(self, X, y=None):
+        """Does nothing: this transformer is stateless.
+        Parameters
+        ----------
+        X : cudf.Series(A Series of string documents)
+        """
+        if not (
+            isinstance(X, cudf.Series)
+            and isinstance(X._column, cudf.core.column.StringColumn)
+        ):
+            raise ValueError(f"cudf.Series([str]) expected ,got {type(X)}")
+        self._warn_for_unused_params()
+        self._validate_params()
```

Comment:
Technically `fit` does run these methods, so the it would be good for the docstring to mention it

Reply:
Likely Resolved by https://github.com/rapidsai/cuml/pull/2554/commits/145f59aff7996459080439df7395ac912f2b25ff . 
, let me know if looks fine now.